packages/core/index.js

```diff
@@ -9,7 +9,7 @@ const port = 3000
 // This is here to demo intercept
 // Intercept this service and replace this message
 // when running locally
-const where = 'cluster'
+const where = process.env.HOST
```

Comment:
Incase HOST isn't set, we may want a fallback. 
```suggestion
const where = process.env.HOST || 'local'
```
Or , could even consider other environment variables  (ex:  `process.env.PWD`?)